sound/soc/sof/intel/tgl.c

```diff
@@ -39,14 +39,18 @@ static int tgl_dsp_core_get(struct snd_sof_dev *sdev, int core)
 static int tgl_dsp_core_put(struct snd_sof_dev *sdev, int core)
 {
 	const struct sof_ipc_pm_ops *pm_ops = sdev->ipc->ops->pm;
+	int ret;
```

Comment:
@RanderWang why change this now? It is working as expected for IPC3. If this needs to be different for IP4, you probably need a new op, right?

Reply:
and commit subject: "ASoC: SOF: Intel: tgl: ..."

Reply:
o, thanks, it is also used by ipc3